### PR TITLE
Fix DebeziumIO JmsIO PreCommit

### PR DESCRIPTION
**Pull request by Abacn**
For DebeziumIO PreCommit, it's due to https://debezium.io/blog/2023/04/25/container-images-quayio/

For JmsIO PreCommit, it appears tests run on Java11 experiences elevated number of unack message

**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by Abacn** [comment]
found this also affects Python PostCommit

**Comment by Abacn** [comment]
R: @Polber

**Comment by Abacn** [comment]
R: @jrmccluskey

File: sdks/go/test/integration/io/xlang/debezium/debezium_test.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ import (
 )
 
 const (
-	debeziumImage = "debezium/example-postgres:latest"
+	debeziumImage = "quay.io/ebezium/example-postgres:latest"
```
Comment (jrmccluskey) [inline sdks/go/test/integration/io/xlang/debezium/debezium_test.go:37]:
```suggestion
	debeziumImage = "quay.io/debezium/example-postgres:latest"
```

**Comment by jrmccluskey** [comment]
I have a fix for debezium in #32938 that isn't targeted at the new image path, just pinning the version. If you would close #32936 and #32967 as closed by this that's fine

File: sdks/go/test/integration/io/xlang/debezium/debezium_test.go (hunk 1 above)
Comment (Abacn) [reply sdks/go/test/integration/io/xlang/debezium/debezium_test.go]: oops, thanks for catching this

**Pull request merged**